inc/autolinks.php

```diff
@@ -983,7 +983,7 @@ public function taxopress_manage_autolinks()
                                                 </h3>
                                             </div>
                                             <div class="inside">
-                                                <p><?php echo sprintf(esc_html__('If your website language does not use the A-Z alphabet, %1s please read this documentation. %2s', 'simple-tags'), '<a href="https://taxopress.com/docs/characters/">', '</a>'); ?>
```

Comment:
Did you work on this line? If not, why is the line reformatted? Please correct and remove your format